File: Santander/app/src/main/java/com/avanade/santander/fundos/domain/model/Info.java
package com.avanade.santander.fundos.domain.model;

import com.google.gson.annotations.Expose;

public class Info {

    @Expose
    private final String name;
    @Expose
    private final String data;

    public Info(String name, String data) {
        this.name = name;
        this.data = data;
    }

    public String getName() {
        return name;
    }

    public String getData() {
        return data;
    }
}

File: app/src/main/java/com/galdino/testandroid/data/entity/investment/Info.kt
package com.galdino.testandroid.data.entity.investment

import com.google.gson.annotations.SerializedName

data class Info(@SerializedName("name")
                override val name: String? = null,

                @SerializedName("data")
                override val data: String? = null): InfoModel(name, data) {
}